Tests/StarknetTests/Utils/DevnetClient.swift

```diff
@@ -0,0 +1,235 @@
+        isDevnetRunning = true
+    }
+    
+    public func close(){
+        if(!isDevnetRunning){
+            return
+        }
+        
+        devnetProcess.terminate()
+        
+        // Wait for the process to be terminated
+        devnetProcess.waitUntilExit()
+        isDevnetRunning = false
+    }
+    
+    // needs finishing
+    public func prefundAccount(accountAddress: Felt) {
+        let url = URL(string: "https://httpbin.org/post?address=\(accountAddress)&amount=5000000000000000")
+        guard let requestUrl = url else { fatalError() }
+        // Prepare URL Request Object
+        var request = URLRequest(url: requestUrl)
+        request.httpMethod = "POST"
+         
+        // HTTP Request Parameters which will be sent in HTTP Request Body
+        let postString = "{\"address\":\"\(accountAddress.toHex())\",\"amount\": 5000000000000000}"
+        // Set HTTP Request Body
+        request.httpBody = postString.data(using: String.Encoding.utf8);
+        // Perform HTTP Request
+        let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
+                
+                // Check for Error
+                if let error = error {
+                    print("Error took place \(error)")
+                    return
+                }
+         
+                // Convert HTTP Response Data to a String
+                if let data = data, let dataString = String(data: data, encoding: .utf8) {
+                    print("Response data string:\n \(dataString)")
+                }
+        }
+        task.resume()
+    }
+    
+    public func deployAccount(name: String) {
+        let params = [
+            "--account_dir",
+            accountDirectory,
+            "--account",
+            name,
+            "--wallet",
+            "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount"]
+        
+        let _ = runStarknetCli(
+                name: "Create account config",
+                command: "new_account",
+                args: params.joined(separator: " "))
+        
+        //TODO prefundAccount
+        let _ = runStarknetCli(
+                        name: "Account deployment",
+                        command: "deploy_account",
+                        args: params.joined(separator: " "))
+    }
+    
+    private func runStarknetCli(name: String, command: String, args: String) -> String {
+        let process = Process()
+        
+        let outputPipe = Pipe()
+        let errorPipe = Pipe()
+        process.standardOutput = outputPipe
+        process.standardError = errorPipe
+        process.arguments = [
+            "-l",
+            "-c",
+            "/Users/jakub/.asdf/shims/starknet \(command) \(args) --gateway_url \(gatewayUrl) --feeder_gateway_url \(feederGatewayUrl) --network alpha-goerli"]
+        
+        process.launchPath = "/bin/zsh"
+        process.standardInput = nil
+        process.launch()
+        process.waitUntilExit()
```

Comment:
I think we should make runStarknetCli explicitly async, and wrap this in a Task, which we can then await. But hence it's only in tests, we can keep it this way for now.